lib/honeybadger/trace.rb

```diff
@@ -45,8 +46,10 @@ def add_query(event)
       end
     end
 
-    def complete(event)
+    def complete(event, config)
       @meta = clean_event(event).to_h
+      @meta[:params] = Util::Sanitizer.new(filters: config.params_filters).sanitize(event.payload[:params]) if event.payload[:params]
```

Comment:
Yes, event.payload[:params] is controller params.  Feel free to send the additional data if you want. :)